packages/react/src/ReactChildren.js

```diff
@@ -25,6 +25,12 @@ var FAUX_ITERATOR_SYMBOL = '@@iterator'; // Before Symbol spec.
 var REACT_ELEMENT_TYPE =
   (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.element')) ||
   0xeac7;
+const REACT_CALL_TYPE =
+  (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.call')) ||
+  0xeac8;
+const REACT_RETURN_TYPE =
+  (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.return')) ||
+  0xeac9;
 const REACT_PORTAL_TYPE =
   (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.portal')) ||
```

Comment:
Does it make sense to share the `typeof Symbol === 'function' && Symbol.for)` here? in a `supportsSymbolFor` variable, or something?

Reply:
Doesn't matter right now. This code executes once. Although it probably makes sense to copy how we do it in `ReactChildFiber`.